axolotl-web/src/pages/MessageList.vue

```diff
@@ -22,91 +22,84 @@
         </div>
           <div v-for="(message) in messageList.Messages.slice().reverse()"
               :class="{'col-12':true,
-                      'sent':message.Outgoing,
-                      'reply':!message.Outgoing,
+                      'outgoing': message.Outgoing,
+                      'sent':message.IsSent && message.Outgoing,
+                      'received':message.Receipt && message.Outgoing,
```

Comment:
This should be message.IsRead because Receipt is not used at all

Reply:
Fixed.